packages/palette/src/elements/DataVis/ChartTooltip.tsx

```diff
@@ -41,9 +53,14 @@ export function isChartTooltipProps(obj: any): obj is ChartTooltipProps {
 export const ChartTooltip = ({
   title,
   description,
+  noPadding,
   ...others
 }: ChartTooltipProps) => (
-  <ChartTooltipWrapper {...others}>
+  <ChartTooltipWrapper
+    px={noPadding ? 0 : 0.5}
+    py={noPadding ? 0 : 1}
```

Comment:
ah, you inverted these values. when theres padding, py should be 0.5 and px should be 1.

Reply:
🙀